lighthouse-plugin-publisher-ads/audits/ad-render-blocking-resources.js

```diff
@@ -74,7 +74,7 @@ class AdRenderBlockingResources extends RenderBlockingResources {
       id: 'ad-render-blocking-resources',
       title: str_(UIStrings.title),
       failureTitle: str_(UIStrings.failureTitle),
-      scoreDisplayMode: 'numeric',
+      scoreDisplayMode: 'binary',
```

Comment:
I also toyed with removing line 122 below, but it also did not make any visual difference.